src/content/docs/analyzer/suppressions.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: Suppressions
+description: How to use suppressions.
+---
+
+The Biome analyzer is the foundation of the [linter](/linter) and [assist;](/assist) in fact, both tools share a lot of similarities.
+
+Among them, they share the same suppression engine, which means that you can suppress a lint rule the same way you would suppress an assist action.
+
+With suppression, it's possible to turn off a lint rule (or action) for a specific line of the code, more lines of code, or the entire file.
+
+Suppression is achievable with **suppression comments**.
+
+## Suppression syntax
+
+Suppression comments have the following format:
+
+```js
+// biome-ignore lint: <explanation>
+// biome-ignore assist: <explanation>
+// biome-ignore syntax: <explanation>
+// biome-ignore lint/suspicious: <explanation>
+// biome-ignore lint/suspicious/noDebugger: <explanation>
+// biome-ignore lint/suspicious/noDebugger(foo): <explanation>
+// biome-ignore-all lint: <explanation>
+// biome-ignore-start lint: <explanation>
+// biome-ignore-end lint: <explanation>
+```
+
+Let's break it down:
```

Comment:
The part that follows felt a bit off to me, since we're not actually breaking down a single example. So I've rephrased a bit below.